File: KIFSwiftPMConsumer/KIFSwiftPMConsumer/ViewController.m
//
//  ViewController.m
//  KIFSwiftPMConsumer
//
//  Created by Derek Ostrander on 3/1/21.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *testLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.testLabel.accessibilityLabel = @"Test Label";
}


@end

File: Documentation/Examples/SPMIntegration/SPMIntegration/ViewController.swift
//
//  ViewController.swift
//  SPM Integration
//
//  Created by Pablo Bartolome on 24/3/21.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var label: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        label.text = "Waiting..."
    }

    @IBAction func didTapButton(_ sender: Any) {
        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
            self.label.text = "Done"
        }
    }
}

